internal/synthetics/test.yml

```diff
@@ -0,0 +1,56 @@
+config:
+  batchName: Test Batch Name
+  branch: sample-branch
+  commit: f749615d0e3f37d6d9f1ebf5a707359448fd644b
+  deepLink: https://test-site.com
+  platform: test-platform
+  repository: sample-repository
+tests:
+  - config:
+      isBlocking: true
+    #      overrides:
```

Comment:
Just for my understanding, did we just commented out (overrides) since API is unstable at the moment and this is sth we plan to reinstate back once API issue has been resolved ?

Reply:
Yeah 

Just an FYI (for others who find this comment too) - since there exists an issue with the API when the `overrides` attribute is specified, the `overrides` attribute in this YAML file has been commented, and can be uncommented after the API is stable and throws no errors upon specifying the `overrides` attribute